crates/foundry-ui/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+pub use message::*;
+
+pub mod message;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
+pub enum OutputFormat {
+    #[default]
+    Human,
+    Json,
+}
+
+impl OutputFormat {
+    #[must_use]
+    pub fn from_flag(json: bool) -> Self {
+        if json {
+            OutputFormat::Json
+        } else {
+            OutputFormat::Human
+        }
+    }
+}
```

Comment:
note: I know that flags responsible for output format is currently only present in sncast, but thinking ahead it's reasonable to have it at generic ui level.